package.json

```diff
@@ -44,7 +45,10 @@
     "@vueuse/core": "^10.7.2",
     "@vueuse/nuxt": "^10.7.2",
     "ajv": "^8.12.0",
+    "fs": "^0.0.1-security",
+    "glob": "^10.3.10",
     "lodash": "^4.17.21",
+    "path": "^0.12.7",
```

Comment:
why do you need these?

Reply:
I think this showed up when I did npm i geode, should I delete them?

Reply:
try to delete them